src/main/java/seedu/address/model/person/Task.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.model.person;
+
+import java.util.Objects;
+
+/**
+ * Represents an Item's Task in ELISA.
+ * Completion state of Event is false by default.
+ * Priority of Event is medium by default.
+ * Guarantees: immutable;
+ */
+public class Task {
+
+    private final Priority priority;
+    private final Boolean complete;
+
+    /**
+     * Constructs a {@code Task}.
+     *
+     * @param priority A Priority of the event. Defaults to Priority.MEDIUM if null.
+     * @param complete Denotes whether the task has been completed or not. Defaults to false if null.
+     */
+    public Task(Priority priority, Boolean complete) {
+
+        if (priority != null) {
+            this.priority = priority;
+        } else {
+            this.priority = Priority.MEDIUM;
+        }
+
+        if (complete != null) {
+            this.complete = complete;
+        } else {
+            this.complete = false;
+        }
+    }
+
+    public Priority getPriority() {
```

Comment:
Perhaps you can create a setPriority() method also so that the user can edit the priority level?